From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 29 Apr 2020 15:35:30 +0200
Subject: ANDROID: vfs: fix export symbol type

In commit 044366659c18 ("ANDROID: vfs: Add setattr2 for filesystems with
per mount permissions") a new symbol was exported, but it should have
been set as a _GPL symbol.

Fix this up by properly.

Bug: 35848445
Cc: Daniel Rosenberg <drosen@google.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I87585ad059367aa51b784ec415a1bf7f809de769
---
 fs/attr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/attr.c b/fs/attr.c
--- a/fs/attr.c
+++ b/fs/attr.c
@@ -347,7 +347,7 @@ int notify_change2(struct vfsmount *mnt, struct dentry * dentry, struct iattr *
 
 	return error;
 }
-EXPORT_SYMBOL(notify_change2);
+EXPORT_SYMBOL_GPL(notify_change2);
 
 int notify_change(struct dentry * dentry, struct iattr * attr, struct inode **delegated_inode)
 {
